src/Libraries/CoreNodeModels/Input/CustomSelection.cs

```diff
@@ -0,0 +1,291 @@
+using Autodesk.DesignScript.Runtime;
+using Dynamo.Graph;
+using Dynamo.Graph.Nodes;
+using Newtonsoft.Json;
+using ProtoCore.AST.AssociativeAST;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Windows.Input;
+using System.Xml;
+using CoreNodeModels;
+using CoreNodeModels.Properties;
+
+
+namespace CoreNodeModels.Input
+{
+    [NodeName("Custom Dropdown")]
+    [NodeCategory(BuiltinNodeCategories.CORE_INPUT)]
+    [NodeDescription("CustomSelectionNodeDescription", typeof(Properties.Resources))]
+    [NodeSearchTags("CustomSelectionSearchTags", typeof(Properties.Resources))]
+    [OutPortNames("value")]
+    [OutPortTypes("object")]
```

Comment:
I think `var` or `var[]..[]` is more appropriate depending on what this node does.